SocialCareCaseViewerApi/V1/Gateways/ProcessDataGateway.cs

```diff
@@ -33,11 +33,11 @@ public ProcessDataGateway(ISccvDbContext sccvDbContext, DatabaseContext database
             FilterDefinition<BsonDocument> firstNameFilter;
             FilterDefinition<BsonDocument> lastNameFilter;
 
-            if (!string.IsNullOrWhiteSpace(request.MosaicId))
+            if (!string.IsNullOrWhiteSpace(request.RecordId))
```

Comment:
This is just a temporary solution until the proper get end point is in place, so for now please update this so that the RecordId takes precedence over MosaicId.

Reply:
If recordId is used, you can simply return the result from within this if statement since none of the later filters (dates etc.) apply